app/migrations/0002_auto_20210406_1557.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.2 on 2021-04-06 15:57
+
```

Comment:
Do these files needed to be added to the Gitignore or are they needed in the merge?